<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickUrlWindow</class>
 <widget class="QDialog" name="QuickUrlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickUrlWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="memeTypeLabel">
       <property name="text">
        <string>Meme type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="urlInput"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="optionalLabel">
       <property name="text">
        <string>Optional new filename</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="optionalInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
